webpack.config.js

```diff
@@ -25,7 +25,7 @@ const autoprefixer = require('autoprefixer');
 
 const env = process.env.NODE_ENV;
 
-const THIS_APP_ID = 'orderentry';
+const THIS_APP_ID = 'openmrs-owa-orderentry';
```

Comment:
Why does the name change any time you make a commit?

Reply:
I'm yet to figure this out.

Reply:
Ok. cool.